autospec/build.py

```diff
@@ -240,3 +240,16 @@ def package(filemanager, mockconfig, mockopts, cleanup=False):
         exit(1)
 
     parse_build_results(download_path + "/results/build.log", returncode, filemanager)
+
+    files = os.listdir('{}/results'.format(download_path))
+
+    os.makedirs('{}/results/srpm'.format(download_path))
```

Comment:
What about when someone already has these directories on their system from a previous run? This will fail with a traceback. You should capture `FileExistsError` exceptions and ignore them.

Reply:
I wrongly assumed it makedirs works like mkdir -p :) fixing.